src/ValueObject/PhpVersionFeature.php

```diff
@@ -161,6 +161,16 @@ final class PhpVersionFeature
      */
     public const BEFORE_OBJECT_TYPE = '7.1';
 
+    /**
+     * @var string
+     */
+    public const BEFORE_VOID_RETURN_TYPE = '7.0';
+
+    /**
+     * @var string
+     */
+    public const BEFORE_NULLABLE_TYPE = '7.0';
```

Comment:
I wonder, I have no idea why I choose string type for PHP Versions. Maybe based on `composer.json`

I wonder, is there a better type to handle PHP versions here?

E:g.

- `70400`
- 7.4 (float)

<br>

This way here could re-used another numeric value:

```php
public const BEFORE_NULLABLE_TYPE = NULLABLE_TYPE - 0.1;

// 0.1 could be also constant
public const BEFORE_NULLABLE_TYPE = NULLABLE_TYPE - MINOR_BUMP; 
```


What do you think?

Reply:
`70400` <= I like it, it simplifies the comparison, and you can directly check against `PHP_VERSION_ID`.

Better doing `-1` instead of `-0.1` then it's always an `int`:

```php
public const BEFORE_NULLABLE_TYPE = NULLABLE_TYPE - 1;
```

Reply:
Even better :+1: I'll make na issue